internal/config/config.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 
 	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+	rhiconfig "github.com/redhatinsights/app-common-go/pkg/api/v1"
```

Comment:
does it make sense to just import this package once as rhiconfig rather than doing it twice with two aliases?